pkgs/applications/science/machine-learning/starspace/default.nix

```diff
@@ -1,8 +1,8 @@
-{ lib, stdenv, fetchFromGitHub, boost165, zlib }:
+{ lib, stdenv, fetchFromGitHub, boost, zlib }:
 
 stdenv.mkDerivation rec {
   pname = "starspace";
-  version = "unstable-2021-01-17";
+  version = "unstable-2019-12-13";
```

Comment:
Hmm 2021-01-17 was the date the package was created, not the date of the last commit of the repo. Very unfortunate. Not sure it's worth changing it?

Reply:
According to https://nixos.org/manual/nixpkgs/stable/#sec-package-naming:

> If a package is not a release but a commit from a repository, then the version attribute must be the date of that (fetched) commit. The date must be in "unstable-YYYY-MM-DD" format.

So I guess yes.

Reply:
Fair enough. Though I'd say put it in a separate commit at least?

Reply:
Done.